src/dune_pkg/feature_flags.ml

```diff
@@ -1,4 +1,4 @@
 open! Import
 
-let use_toolchains = false
-let use_autorelock = false
+let use_toolchains = true
+let use_autorelock = true
```

Comment:
You mentionned that it's not necessary, but let's not forget to remove this. Also, the tests should work without these flags.
(note to self: we really need a way to enable that at runtime or compile time, since we're bound to forget at some point)

Reply:
I agree.

Reply:
yes, I'll take care of this

Reply:
I think a better way to go about it could be to target `main`. The log printing happens there as well and adding the PR to `toolchains` at this point only to upstream toolchains and having to upstream this afterwards anyway seems like it would require dealing with landing this PR essentially twice.

Reply:
ah I had missed this. Yes, this should target `main`.

Reply:
Ok, I can rebase the work against `main` then

Reply:
you'll need to change the base branch of the PR too (edit button next to title)